opentaxii/middleware.py

```diff
@@ -73,7 +73,9 @@ def wrapper(relative_path=""):
             if service:
                 if (service.authentication_required
                         and context.account is None):
-                    raise UnauthorizedException()
+                    raise UnauthorizedException(
+                        status_type=server.config['unauthorized_status'],
```

Comment:
it's unclear to me what values are accepted for `status_type`, do you have a source for this information?

Reply:
Yes, `libtaxii.constants.ST_TYPES_11`. I've updated the docs.